extensions/wikia/AdEngine/js/lookup/openXBidder.js

```diff
@@ -120,7 +147,7 @@ define('ext.wikia.adEngine.lookup.openXBidder', [
 		for (slotName in prices) {
 			if (prices.hasOwnProperty(slotName) && prices[slotName].price !== priceTimeout) {
 				shortSlotName = adSlot.getShortSlotName(slotName);
-				priceMap[shortSlotName] = prices[slotName].price;
+				priceMap[shortSlotName] = prices[slotName];
```

Comment:
Did you check how does it work with our GA tracking? I think there may be something wrong in:
`openXBidder::encodeParamsForTracking` and tracking `lookup_end` event
I'm not sure about `lookupFactory::trackState` and `lookup_success`/`lookup_error` events